assessment_software/src/app/page/components/components.component.html

```diff
@@ -1,3 +1,4 @@
+<app-navbar></app-navbar>
```

Comment:
- Remove this line and add it in app.component.html.
- remove <p>Hello World</p> from app.component.html. 

```
<app-navbar></app-navbar>
<router-outlet></router-outlet>
```

in the nav bar, add archived, courses and component page
```
  title:string[]=['Courses','Archived','Components'];
  url:string[]=["\courses","\archived","\component"];
```

and make sure the avatar drop-down works and the page change password is also connected. 

Then, you will see how this is working.